dds_web/database/models.py

```diff
@@ -124,6 +123,18 @@ def safespring_project(self):
 
         return self.responsible_unit.safespring
 
+    @property
+    def size(self):
+        """Calculate size of project."""
+
+        return sum([f.size_stored for f in self.files])
```

Comment:
I recall there might be a way to do the sum in SQL. There might be some magic in this line, but to me it looks like the sum is done in pure python. I think it's possible to use `func.sum`. This is an optimization though so it might not be needed.

Reply:
Yeah, there is, but that means you have to perform a query and then anyway get the list of file ids in `self.files` I think. But we can definitely look at it later on.